src/app/components/XProfileWidget/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import './XProfileWidget.css'
```

Comment:
Is this intentionally not a css module?

Reply:
(so far we don't even have css modules in this repo; but global css is even more out of place)

Reply:
I have turned this into a CSS module.